README.md

```diff
@@ -41,10 +41,19 @@ fastify
     if (err) console.error(err)
 
     console.log(fastify.config) // or fastify[options.confKey]
+    console.log(fastify.getEnvs())
     // output: { PORT: 3000 }
   })
 ```
 
+You can also use the function `getEnvs()` of the Request from within a handler function: 
+```js
+fastify.get('/', (request, reply) => {
+    console.log(request.getEnvs())
+    // output: { PORT: 3000 }
+})
+```
+
```

Comment:
```suggestion
```

Note that the `getEnvs` decorators will not be added if they already exist.

```

Reply:
Done.